app/assets/javascripts/registration/register.js

```diff
@@ -77,28 +77,28 @@ function DorRegistration(initOpts) {
         beforeSend: function(xhr) {
           $t.setStatus(data, 'pending')
         },
-        dataType: 'json'
+        dataType: 'json',
+        success: function(response,status,xhr) {
```

Comment:
I see this is ordered / grouped differently.

Reply:
No, it's refactored because xhr no longer has a `success` method.